orca-core/src/test/groovy/com/netflix/spinnaker/orca/pipeline/persistence/ExecutionRepositoryTck.groovy

```diff
@@ -566,16 +574,103 @@ class JedisExecutionRepositorySpec extends ExecutionRepositoryTck<JedisExecution
     100   || 4
   }
 
+  def "can store an orchestration already in previousRedis back to previousRedis"() {
```

Comment:
Is that actually what we want to do? Don't we want to migrate it to the new Redis?

Reply:
Migration is still the work of `MultiRedisOrchestrationMigrationNotificationAgent` and `MultiRedisPipelineMigrationNotificationAgent`, which only operate over completed executions. We'll want to rewrite the migration process once `DynomiteExecutionRepository` is merged in with schema changes and I'd like to tackle migrating running executions then too.

Reply:
Ok, cool